src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx

```diff
@@ -126,6 +126,15 @@ const GroupDetails = createReactClass({
           Sentry.withScope(scope => {
             Sentry.captureException(new Error('Project not found'));
           });
+        } else {
+          const locationWithProject = {...browserHistory.getCurrentLocation()};
```

Comment:
Same thing here, since `GroupDetails` is a route endpoint, it will have router props passed to it.

Reply:
Done